organisations/admin.py

```diff
@@ -74,7 +75,7 @@ def __init__(self, *args, **kwargs):
 
 
 @admin.register(OrganisationProposal)
-class OrganisationProposalAdmin(admin.ModelAdmin):
+class OrganisationProposalAdmin(AuditlogAdminViewAccessLogMixin, admin.ModelAdmin):
```

Comment:
As OrganisationProposalAdmin's list_display contains name which is the applicant's name and GDPR data, wouldn't it make sense to log the list views because of this?
```suggestion
class OrganisationProposalAdmin(AuditlogAdminViewAccessLogMixin, admin.ModelAdmin):
    enable_list_view_audit_logging = True
```

Reply:
This is just spam to auditlog. Organisation names are public anyway. It's only admins who are reading them from django admin site.

<img width="678" alt="image" src="https://github.com/user-attachments/assets/2ccd5aa6-0a11-4481-a6d6-a9f4b997ef65" />

Reply:
After discussion this point can be left as is, or masked, if wanted to, and can be possibly iterated on later (not in this PR).